### PR TITLE
[WIP] Webapp data plotting

### PR DESCRIPTION
**Pull request by cliffckerr**
WIP implementation of data plotting for the webapp. Still trying to tag you @deviozc, @sayers24

Closes https://github.com/InstituteforDiseaseModeling/covasim/issues/78, https://github.com/InstituteforDiseaseModeling/covasim/issues/79

### CODE REVIEW

File: covasim/webapp/example_web_data.csv [hunk 1]
```diff
@@ -0,0 +1,32 @@
+date,new_diagnoses,new_tests,new_deaths
+2020-03-01,1,24,0
```
Comment (sayers24) [inline covasim/webapp/example_web_data.csv:2]: Do we need a calibration date in the UI? Seem like right now, everything is based on n days from "start". Though, that could be a separate task if this is assuming the first entry is day 0

**Comment by lgeorge-idm** [comment]
@ckerr-IDM, you assigned me to this PR. What needs to be done here?

**Comment by ckerr-IDM** [comment]
@lgeorge-idm I think the plotting is OK, but at the moment it's just loading a dummy data file, which is causing the tests to break from a path issue -- so actually the part of this that still needs to be done is just https://github.com/InstituteforDiseaseModeling/covasim/issues/79 (just tagged you in that)

File: covasim/webapp/example_web_data.csv (hunk 1 above)
Comment (ckerr-IDM) [reply covasim/webapp/example_web_data.csv:2]: @sayers24 yes, date rather than day index would be great!

**Pull request closed without merging**